PyDictionary/core.py

```diff
@@ -135,7 +135,7 @@ def meaning(term, disable_errors=False):
                 return out
             except Exception as e:
                 if disable_errors == False:
-                    print("Error: The Following Error occured: %s" % e)
+                    raise e
```

Comment:
what type of exceptions would get raised here? is it possible to raise other named exceptions in this block?

Reply:
I haven't been on GitHub in a while. Depends on the error in the Exception. If anyone knows the complete list of errors thrown by this exception, go ahead and list them here.